test/functional/rpc_rawtransaction.py

```diff
@@ -335,6 +339,10 @@ def createrawtransaction_tests(self):
             self.nodes[2].createrawtransaction(inputs=[{'txid': TXID, 'vout': 9}], outputs=[{address: 99}, {address2: 99}, {'data': '99'}]),
         )
 
+        rawtx_v3 = self.nodes[2].createrawtransaction(inputs=[{'txid': TXID, 'vout': 9}], outputs=OrderedDict([(address, 99), (address2, 99)]), version=3)
+        tx = tx_from_hex(rawtx_v3)
+        assert_equal(tx.version, 3)
```

Comment:
Shouldn't this test be repeated for each valid `tx.version` in the range `[TX_MIN_STANDARD_VERSION, TX_MAX_STANDARD_VERSION]`?